Feature Branch: Don't ask which ElasticSearch version to use. Only ES6 is supported from now on.

leonardr opened the pull request:
I successfully built [this circulation branch](https://github.com/NYPL-Simplified/circulation/tree/remove-es1) with this Docker branch and the following command:

`docker build --build-arg version=remove-es1 -f Dockerfile.webapp --no-cache .`

I verified that Python packages for ES6 were installed as part of the build process.

I don't know enough about docker-compose to test that part.

leonardr committed:
I believe this will make Docker always and exclusively use ES6.

jonathangreen commented:
I'll take a look at this one today. Did the branch on the circulation manager get deleted? I get a 404 when following your link.

leonardr commented:
Yeah, sorry about that. Try building the feature branch: https://github.com/NYPL-Simplified/circulation/tree/avoid-materialized-view-in-special-lanes

jonathangreen commented on docker-compose.yml:
```diff
@@ -9,7 +9,7 @@ services:
       - "dbdata:/var/lib/postgresql/data"
   es:
-    image: "elasticsearch:1"
+    image: "elasticsearch:6"
```
This won't work since there isn't a tag for 6 in the elasticsearch docker hub. Maybe change it to: 
```yaml
image: "elasticsearch:6.8.1"
```

jonathangreen reviewed (approved):
Other then updating the ES version in the docker compose file, this looks good to me. 👍🏻

leonardr committed:
Run the search_index_clear script in the middle of the night. This will guarantee a fresh build of the index every night.

leonardr committed:
Use an Elasticsearch image that's actually in the Elasticsearch Docker hub.

leonardr commented:
I've changed the elasticsearch tag to the one you suggested. I also changed the crontab to run the script defined in https://github.com/NYPL-Simplified/circulation/pull/1269.
